src/extension.ts

```diff
@@ -78,6 +78,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 			outputChannel
 		});
 	});
+	const removeDiagnosticsOnActiveFile = vscode.commands.registerCommand(Constants.COMMAND_REMOVE_DIAGNOSTICS_ON_ACTIVE_FILE, (selection: vscode.Uri, multiSelect?: vscode.Uri[]) => {
+		return _clearDiagnosticsForSelectedFiles(multiSelect && multiSelect.length > 0 ? multiSelect : [selection], {
+			commandName: Constants.COMMAND_REMOVE_DIAGNOSTICS_ON_ACTIVE_FILE,
+			diagnosticCollection,
+			outputChannel
+		});
+	});
+	const removeDiagnosticsOnSelectedFile = vscode.commands.registerCommand(Constants.COMMAND_REMOVE_DIAGNOSTRICS_ON_SELECTED_FILE, async (selection: vscode.Uri, multiSelect?: vscode.Uri[]) => {
```

Comment:
I see line 81 doesn't have async but this does. Is this intentional?

Reply:
No, it wasn't intentional. But I do not know why async is needed here and I see that with other commands as well. I have added it for now, but I think it doesn't really matter.